HISTORY.rst

```diff
@@ -6,6 +6,8 @@ History
 ------------------
 * Add wrapper for ducc0.wgridder (:pr:204`)
 * Correct Irregular Grid nesting in BeamAxes (:pr:`217`)
+* Add Perley Polyhedron Faceting Gridder/Degridder (:pr:`202`, :pr:`215`)
```

Comment:
Could you start a new version section in the changelog that also mentions this PR? e.g.:

```
0.2.7 (YYYY-MM-DD)
----------------------------
* Add Perley Polyhedron Faceting Gridder/Degridder (:pr:`202`, :pr:`215`, :pr:`222`)

```